dev/reference/sql/functions-and-operators/window-functions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Window Functions
+summary: This document introduces window functions supported in TiDB.
+category: reference
+---
+
+# Window Functions
+
+The usage of window functions in TiDB is similar to that in MySQL. For details, see [MySQL Window Functions](https://dev.mysql.com/doc/refman/8.0/en/window-functions.html). 
+
+Because window functions may use some reserved or key words, some SQL statements that could be executed normally cannot be parsed after TiDB is upgrade. In this case, you can set `tidb_enable_window_function` (`1` by default) to `0`.
```

Comment:
The reserved/keywords is not a possibility. The 'may' is if it impacts your application.

```suggestion
Because window functions reserve additional words in the parser, TiDB provides an option to disable window functions. If you receive errors parsing SQL statements after upgrading, try setting `tidb_enable_window_function=0`.
```